module/chunks/chunkVerifier.go

```diff
@@ -198,6 +201,20 @@ func (fcv *ChunkVerifier) verifyTransactionsInContext(context fvm.Context, chunk
 		return nil, nil, fmt.Errorf("cannot calculate events collection hash: %w", err)
 	}
 	if chunk.EventCollection != eventsHash {
+
+		for i, event := range events {
+
+			fcv.logger.Debug().Int("list_index", i).
+				Str("event_type", string(event.Type)).
```

Comment:
better to add event ID itself, because if the hashing has problem (for instancen, non deterministic), then even if events are the same, but the eventsHash will still be different.

Reply:
Added both ID and fingerprint

Reply:
better to include result ID and the chunk index, otherwise it's not easy to find those logs. 

Since the error contains result ID, we can filter all these debug logs with the result ID.

Reply:
we have block and collection ID - isn't that enough to get events on the EN side?

Reply:
makes sense 👍